templates/statefulset.yaml

```diff
@@ -16,8 +16,22 @@ spec:
     metadata:
       labels:
         app: vaultwarden
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha1sum }}
```

Comment:
I like the idea of adding `checksum/config` and `checksum/secret` to trigger a restart whenever the configs are updated. However, these are `metadata` annotations and not labels:

```yaml
...
    metadata:
      labels:
        <>
      annotations:
        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha1sum }}
        checksum/secret: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha1sum }}
...
```

Thanks!

Reply:
@guerzon Thank you for pointing that out. You are right. It should be in annotations. Fixed it in last commits :)